hack/licence-detector/validate/validate.go

```diff
@@ -0,0 +1,135 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package validate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"runtime"
+	"sync/atomic"
+	"time"
+
+	"github.com/elastic/cloud-on-k8s/hack/licence-detector/dependency"
+	"golang.org/x/sync/errgroup"
+)
+
+const maxErrors = 12
```

Comment:
Why do we tolerate 12 errors? Maybe add a comment.

Reply:
It's for circuit-breaking because in practice we should only ever see a small number of errors. I have added a comment to clarify.